<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alarme</class>
 <widget class="QDialog" name="alarme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tv'Ease</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/img/images/logo.png</normaloff>:/img/images/logo.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #fdf1f1 ;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>310</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>570</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SS</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Saa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>460</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>260</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CHECK WORK</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>520</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>160</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>460</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>570</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>560</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>dqs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>dqd</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>FAS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ZZ</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DDD</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_9">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>570</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>410</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>360</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>360</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>set alarm 5</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dd_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>POUSE</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="alarmTime_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>410</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #333770 ; </string>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>520</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarme :</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>640</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton
{
border-radius : 10px ;
color : white ; 
padding:10px ;
border:none ; 
background-color:#333770;
color : white ; 
}

#pushButton:hover
{
background-color:  ;
}</string>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
